models/iaf_cond_alpha_mc_kinetics.h

```diff
@@ -0,0 +1,651 @@
+/*
+ *  iaf_cond_alpha_mc_kinetics.h
```

Comment:
I am not sure `_kinetics` is an optimal name, since it is unspecific: what kinetics?